setup.py

```diff
@@ -25,6 +25,11 @@
 #
 # Environment variables for feature toggles:
 #
+#   DEBUG_CUDA=1
+#     if used in conjunction with DEBUG or REL_WITH_DEB_INFO, will also
+#     build CUDA kernels with -g -lineinfo --source-in-ptx.  Note that
+#     on CUDA 12 this may cause nvcc to OOM, so this is disabled by default.
+#
```

Comment:
```suggestion
#   DEBUG_CUDA=0
#     if used in conjunction with DEBUG or REL_WITH_DEB_INFO, will
#     build CUDA kernels without -lineinfo --source-in-ptx, to prevent CUDA
#     -12 compiler OOMS/crashes.
```